src/components/include/policy/policy_external/policy/policy_manager.h

```diff
@@ -155,6 +156,14 @@ class PolicyManager : public usage_statistics::StatisticsManager {
    */
   virtual void KmsChanged(int kilometers) = 0;
 
+  /**
+   * @brief Returns state of the lock screen that could be able to be dismissed
+   * while connected to SDL, allowing users the ability to interact with the
+   * app.
+   * @return bool True if lock screen is able to be dismissed.
```

Comment:
@LuxoftAKutsan `can be dismissed` ?

Reply:
ab13e237e12c0be7ad93647e256da0ea449ea2b8